src/lib/components/WorkstreamDetail.svelte

```diff
@@ -33,64 +60,148 @@
-                  <Button variant="primary" icon={ThumbsDown}>Deny</Button>
-                {/if}
+            {/if}
+            {#if !creator}
+              <Tooltip value={applied ? "You've already applied" : null}>
                 <Button
+                  disabled={applied}
+                  icon={Apply}
                   on:click={() =>
-                    modal.show(ApplicationModal, undefined, {
-                      workstream,
-                      application
-                    })}>View</Button
+                    modal.show(ApplyModal, undefined, { workstream })}
                 >
-              </div>
-            </ActionRow>
-          {/each}
-        {/if}
-      </div>
-    </Card>
+                  Apply for {workstream.type}
+                </Button>
+              </Tooltip>
+            {/if}
+          </div>
+        </div>
+      </Card>
+    {:else}
+      <Card hoverable={false}>
+        <div slot="top">
+          <h3 style="margin-bottom: 1rem;">Active stream</h3>
+          <div class="timerate">
+            <div style="text-align: right;">
+              <Rate
+                rate={workstream.payment.rate}
+                currency={workstream.payment.currency}
+              />
+            </div>
+            <div>
+              <p class="timeframe">Active since Jan 5, 2022</p>
+            </div>
+          </div>
+        </div>
+        <div slot="bottom">
+          <ActionRow>
+            <div slot="left">
+              <User address={activeApplication.creator} />
+            </div>
+            <div slot="right" class="row-actions">
+              <p class="proposal">4020 DAI left (8.04 days)</p>
+              <Button
+                on:click={() =>
+                  modal.show(ApplicationModal, undefined, {
+                    workstream,
+                    activeApplication
+                  })}>View application</Button
+              >
+            </div>
+          </ActionRow>
+          <div class="stream-actions">
+            <p>5000 of 8000 DAI topped up</p>
+            <div style="display: flex; gap: .75rem;">
+              <Button variant="primary-outline" icon={Cross}>Pause</Button>
+              <Button>Top up</Button>
+            </div>
+          </div>
+        </div>
+      </Card>
+      <Card hoverable={false} style="margin-top: 1rem;">
+        <div slot="top">
+          <h3>Rejected applications</h3>
+        </div>
+        <div slot="bottom">
+          {#if rejectedApplications}
+            {#each rejectedApplications as application}
+              <ActionRow>
+                <div slot="left">
+                  <User address={application.creator} />
+                </div>
+                <div slot="right" class="row-actions">
+                  {#if application.counterOffer}
+                    <p class="proposal">Rejected</p>
+                  {/if}
+                  {#if $walletStore.connected && $walletStore.address === workstream.creator}
```

Comment:
Should check `$connectedAndLoggedIn` here too.

Reply:
https://github.com/radicle-dev/workstreams-app/pull/45/commits/12e7a4a55c00ee8ee02c15c1c804e3aa0fa38101